src/wrappers/lambda/LambdaExecutor.ts

```diff
@@ -333,6 +335,36 @@ function injectTriggerTags(span: ThundraSpan, pluginContext: PluginContext, exec
     }
 }
 
+function injectAppsyncInfo(request: any, response: any, execContext: ExecutionContext): any {
+    try {
+        const isAppsync = ConfigProvider.get<boolean>(ConfigNames.THUNDRA_LAMBDA_AWS_APPSYNC);
+        ThundraLogger.debug(
+            '<LambdaExecutor> Checked whether AWS Appsync support is enabled for transaction',
+            execContext.transactionId, ':', isAppsync);
+        if (isAppsync) {
+            const traceId = execContext.traceId;
+            if (typeof response === 'object' && response !== null) {
+                const thundraTraceKey = {
+                    trace_id: traceId,
+                };
+                ThundraLogger.debug(
+                    '<LambdaExecutor> Injected Thundra AppSync trace key for transaction',
+                    execContext.transactionId, ':', thundraTraceKey);
+                response[THUNDRA_TRACE_KEY] = thundraTraceKey;
+            } else {
+                ThundraLogger.debug(
+                    '<LambdaExecutor> Since response is not object, \
+                    skipped Thundra AppSync trace key injection for transaction',
+                    execContext.transactionId);
+            }
+
+            InvocationTraceSupport.addOutgoingTraceLink(traceId);
```

Comment:
Should not add trace id as outgoing trace link

Reply:
done.